client/js/main.js

```diff
@@ -71,6 +72,7 @@ parseOptions((function () {
 	parsed.showcol = parsed.showcol || '';
 	parsed.sortcol = (parsed.sortcol || 'phase').toLowerCase();
 	parsed.sortcolorder = (parsed.sortcolorder || '');
+	parsed.crystallisation = parsed.crystallisation || null;
```

Comment:
don't need this it is fine for crystallisation to be undefined.

undefined inputs won't get added to the options object.